packages/application-generic/src/usecases/select-integration/select-integration.usecase.ts

```diff
@@ -26,6 +33,28 @@ export class SelectIntegration {
   async execute(
     command: SelectIntegrationCommand
   ): Promise<IntegrationEntity | undefined> {
+    const isMultiProviderConfigurationEnabled =
+      await this.getFeatureFlag.isMultiProviderConfigurationEnabled(
+        FeatureFlagCommand.create({
+          userId: command.userId,
+          organizationId: command.organizationId,
+          environmentId: command.environmentId,
+        })
+      );
+
+    if (!isMultiProviderConfigurationEnabled) {
+      return this.getDecryptedIntegrationsUsecase.execute(
+        GetDecryptedIntegrationsCommand.create({
+          organizationId: command.organizationId,
+          environmentId: command.environmentId,
+          channelType: command.channelType,
+          findOne: true,
+          active: true,
+          userId: command.userId,
+        })
+      )[0];
+    }
```

Comment:
`isMultiProviderConfigurationEnabled` is not enabled use the "old way" of selecting the integration with `GetDecryptedIntegrationsUsecase`